docs/source/features/create_and_upload_dataset.rst

```diff
@@ -0,0 +1,45 @@
+###########################
+ Create and Upload Dataset
+###########################
+
+This topic describes how to create a dataset and upload it to TensorBay.
+
+****************
+ Create Dataset
+****************
+
+To create a dataset,
+you need to write a **dataloader** and a **catalog** (:ref:`basic_concepts:catalog & subcatalog`), 
+**catalog** is needed only if there is label information inside the dataset.
+
+A **dataloader** is a function to read the original dataset files into a **dataset**.
```

Comment:
Should all the `dataset` that appeared in this chapter be in boldface?

Reply:
It depends on whether the dataset means all the datasets, which contains both dataset and fusion dataset, or means only dataset with data collected from only one sensor.